AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DepthOfFieldAdded.py

```diff
@@ -30,15 +31,141 @@ class Tests:
     depth_of_field_component = (
         "Entity has a DepthOfField component",
         "Entity failed to find DepthOfField component")
-    depth_of_field_disabled = (
+    depth_of_field_component_disabled = (
         "DepthOfField component disabled",
-        "DepthOfField component was not disabled.")
+        "DepthOfField component was not disabled")
+    depth_of_field_component_enabled = (
+        "DepthOfField component enabled",
+        "DepthOfField component was not enabled")
     post_fx_component = (
         "Entity has a Post FX Layer component",
         "Entity did not have a Post FX Layer component")
-    depth_of_field_enabled = (
-        "DepthOfField component enabled",
-        "DepthOfField component was not enabled.")
+    cameraentityid_override_on = (
+        "CameraEntityId Override turned on",
+        "CameraEntityId Override could not be turned on")
+    cameraentityid_override_off = (
+        "CameraEntityId Override turned off",
+        "CameraEntityId Override could not be turned off")
```

Comment:
you can reduce these two to one cameraentityid_override where the tuple ("CameraEntityId Override property set", "CameraEntityId Override property failed to set") then use if for both on/off.
same for most all of these you can have half the tuples to cover maintain